components/learn/TheLearnHeader.vue

```diff
@@ -1,7 +1,6 @@
 <template>
   <ThePageHeader class="the-learn-header">
-    Start your path towards
-    <br>learning
+    Start your path towards learning
```

Comment:
I changed it because for the events page header the `<br>` is not always after the same word, so I think having the size with columns is a good solution. Moreover, I think we can still do the issue you're mentioning. What do you think?

Reply:
@y4izus - I see what you mean. 

Yeah the typewriter effect, used on other pages, is doing some things to the surrounding text, that _maybe_ we didn't anticipate. 

I'm good with allowing this to go in.

Out of curiosity - where'd you get the calculation for the [max-width](https://github.com/Qiskit/qiskit.org/pull/949/files#diff-b40c93ca21d967330c592b9d48faabf7R45) from?

Reply:
As you already know, we are working with the Carbon Design System. It has a [2x Grid](https://www.carbondesignsystem.com/guidelines/2x-grid/overview/) that is the geometric foundation of all the visual elements of IBM Design. If you take a look at the Figma file, you can see these two grids:

<img width="712" alt="Screen Shot 2020-09-23 at 09 55 44" src="https://user-images.githubusercontent.com/17231966/93983518-f539c080-fd82-11ea-85f6-5eaa18271f5d.png">

We count the columns for each size and multiply them for the column's size on this screen size. For example, in the image above, you can count eight columns (white column + red column = 1) for the title, so:
`max-width: 8 * $column-size-large;`

BUT sometimes, some elements do not fit the same way on "carbon design figma" than "carbon design web" 🤷‍♀️. That's why we use a [look & feel](https://ishadeed.com/article/pixel-perfection/) approach, i.e., if you count and code 8 columns, but "your look & feel" is better with 10, go for the 10 ;)